### PR TITLE
Rework Monitoring and Loggins edit forms

### PR DESCRIPTION
**Pull request by dvaldivia**
Screen Shot 2022-01-14 at 11 18 53 AM
Screen Shot 2022-01-14 at 11 20 37 AM

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantLogsModal.tsx [hunk 1]
```diff
@@ -169,7 +175,7 @@ const EditTenantLogsModal = ({
       fieldKey: `diskCapacityGB`,
       required: true,
       value: newDiskCapacityGB as any as string,
-      pattern: /^[0-9]?(10)?$/,
+      pattern: /^[0-9]*$/,
       customPatternMessage: "Must be an integer between 0 and 10",
```
Comment (cniackz) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantLogsModal.tsx:179]: Based on the regular expression, None or any number is accepted. Hence the range integer between 0 and 10 is no longer truth. If we can rephrase it to be accurate, would be nice

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx [hunk 2]
```diff
@@ -134,21 +140,21 @@ const EditTenantMonitoringModal = ({
       fieldKey: `diskCapacityGB`,
       required: true,
       value: newDiskCapacityGB as any as string,
-      pattern: /^[0-9]?(10)?$/,
+      pattern: /^[0-9]*$/,
       customPatternMessage: "Must be an integer between 0 and 10",
```
Comment (cniackz) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:144]: Based on the regular expression, None or any number is accepted. Hence the range integer between 0 and 10 is no longer truth. If we can rephrase it to be accurate, would be nice

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx [hunk 3]
```diff
@@ -134,21 +140,21 @@ const EditTenantMonitoringModal = ({
...
     });
     tenantMonitoringValidation.push({
       fieldKey: `newCPURequest`,
       required: false,
       value: newCPURequest as any as string,
-      pattern: /^[0-9]?(10)?$/,
+      pattern: /^[0-9]*$/,
       customPatternMessage: "Must be an integer between 0 and 10",
```
Comment (cniackz) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:151]: Based on the regular expression, None or any number is accepted. Hence the range integer between 0 and 10 is no longer truth. If we can rephrase it to be accurate, would be nice

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx [hunk 4]
```diff
@@ -134,21 +140,21 @@ const EditTenantMonitoringModal = ({
...
     });
     tenantMonitoringValidation.push({
       fieldKey: `newMemRequest`,
       required: false,
       value: newMemRequest as any as string,
-      pattern: /^[0-9]?(10)?$/,
+      pattern: /^[0-9]*$/,
       customPatternMessage: "Must be an integer between 0 and 10",
```
Comment (cniackz) [inline portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:158]: Based on the regular expression, None or any number is accepted. Hence the range integer between 0 and 10 is no longer truth. If we can rephrase it to be accurate, would be nice

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx (hunk 2 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:144]: previously empty was also a valid match, this preserves that rule

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx (hunk 3 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:151]: previously empty was also a valid match, this preserves that rule

File: portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx (hunk 4 above)
Comment (dvaldivia) [reply portal-ui/src/screens/Console/Tenants/TenantDetails/EditTenantMonitoringModal.tsx:158]: previously empty was also a valid match, this preserves that rule

**Review by cniackz — APPROVED** [review]
Explained on a thread the behavior of the `customPatternMessage` and upper limit has been removed, thanks!

**Review by bexsoft — APPROVED** [review]
